src/main/java/edu/ucsb/nceas/mdq/rest/SuitesResource.java

```diff
@@ -34,64 +34,62 @@
 import edu.ucsb.nceas.mdqengine.model.Suite;
 import edu.ucsb.nceas.mdqengine.serialize.JsonMarshaller;
 import edu.ucsb.nceas.mdqengine.serialize.XmlMarshaller;
+import edu.ucsb.nceas.mdqengine.dispatch.Dispatcher;
 
 /**
  * Root resource (exposed at "suites" path)
  */
 @Path("suites")
 public class SuitesResource {
-	
-	private Log log = LogFactory.getLog(this.getClass());
+
+    private Log log = LogFactory.getLog(this.getClass());
     private static Controller metadigCtrl = null;
-		
-	public SuitesResource() {}
-	
+
+    public SuitesResource() {}
+
     /**
-     * Method handling HTTP GET requests. The returned object will be sent
-     * to the client as "text/plain" media type.
+     * Method handling HTTP GET requests. The returned object will be sent to the client as
+     * "text/plain" media type.
      *
      * @return String that will be returned as a text/plain response.
      */
     @GET
     @Produces(MediaType.APPLICATION_JSON)
     public String listSuites() {
-        boolean persist = true;
+        boolean persist = false;
         MDQStore store = null;
-        MDQEngine engine = null;
         try {
             store = StoreFactory.getStore(persist);
-            engine = new MDQEngine();
-        } catch (MetadigException | IOException | ConfigurationException e) {
+        } catch (MetadigException e) {
             InternalServerErrorException ise = new InternalServerErrorException(e.getMessage());
-            throw(ise);
+            throw (ise);
         }
 
-    	Collection<String> suites = store.listSuites();
-    	store.shutdown();
+        Collection<String> suites = store.listSuites();
+        store.shutdown();
         return JsonMarshaller.toJson(suites);
     }
-    
+
     @GET
     @Path("/{id}")
     @Produces(MediaType.TEXT_XML)
-    public String getSuite(@PathParam("id") String id) throws UnsupportedEncodingException, JAXBException {
-        boolean persist = true;
+    public String getSuite(@PathParam("id") String id)
+            throws UnsupportedEncodingException, JAXBException {
+        boolean persist = false;
         MDQStore store = null;
-        MDQEngine engine = null;
         try {
             store = StoreFactory.getStore(persist);
-            engine = new MDQEngine();
-        } catch (MetadigException | IOException | ConfigurationException e) {
+        } catch (MetadigException e) {
             InternalServerErrorException ise = new InternalServerErrorException(e.getMessage());
-            throw(ise);
+            throw (ise);
         }
-    	Suite suite = store.getSuite(id);
+        Suite suite = store.getSuite(id);
         store.shutdown();
         return XmlMarshaller.toXml(suite, true);
     }
-    
-//    @POST
-//    @Consumes(MediaType.MULTIPART_FORM_DATA)
+
+    // @POST
+    // @Consumes(MediaType.MULTIPART_FORM_DATA)
```

Comment:
These are commented out because they it is part of the API that was implemented, but then later disabled due to security issues. We had planned to reimplement these API endpoints with appropriate security guards, but that work got de-prioritized. In an ideal world, the full API would be functional. @jeanetteclark maybe we should have a comment pointing at tickets describing why these were disabled and what is needed to re-enable them?